script.js

```diff
@@ -1,4 +1,254 @@
+    currentGameMode = "scissors-paper-stone";
+    myOutputValue = `Hello ${userName}! <br>Let's play scissors ✂️- paper 🗒- stone 🥌!<br>To play, enter scissors, paper, or stone, then click submit.`;
+  } else if (currentGameMode == "scissors-paper-stone") {
+    //format the input object name
+    var inputObjectFormatted = function (input) {
+      if (input == "scissors") {
+        return "scissors ✂️";
+      }
+      if (input == "paper") {
+        return "paper 🗒";
+      }
+      if (input == "stone") {
+        return "stone 🥌";
+      }
+    };
+
+    var inputObject = inputObjectFormatted(input);
+
+    // format program output object name
+    var outputObjectFormatted = function (program) {
+      if (program == "scissors") {
+        return "scissors✂️";
+      }
+      if (program == "paper") {
+        return "paper🗒";
+      }
+      if (program == "stone") {
+        return "stone 🥌";
+      }
+
+      return program;
+    };
+
+    var outputObject = outputObjectFormatted(program);
+
+    turnsPlayed = turnsPlayed + 1;
+
+    //game rules - how to win
+    if (
+      (input == "scissors" && program == "paper") ||
+      (input == "paper" && program == "stone") ||
+      (input == "stone" && program == "scissors")
+    ) {
+      userWon = userWon + 1;
+      computerWon = computerWon;
+      draw = draw;
+      percentageWon = (userWon / turnsPlayed) * 100;
+
+      if (percentageWon >= 50) {
+        return (myOutputValue = `You chose ${inputObject} <br> The computer chose ${outputObject}. <br><br>You win! Congrats!<br><br>So far ${userName}, you have been winning ${userWon}/${turnsPlayed} turns (or ${percentageWon}%). Doing well!`);
+      } else if (percentageWon < 50) {
+        return (myOutputValue = `You chose ${inputObject} <br> The computer chose ${outputObject}. <br><br>You win! Congrats!<br><br>So far ${userName}, you have been winning ${userWon}/${turnsPlayed} turns (or ${percentageWon}%). <br>Keep up the winning streak to boost your stats!`);
+      }
+    }
+
+    //game rules - how to lose
+
+    if (
+      (input == "paper" && program == "scissors") ||
+      (input == "stone" && program == "paper") ||
+      (input == "scissors" && program == "stone")
+    ) {
+      userWon = userWon;
+      computerWon = computerWon + 1;
+      draw = draw;
+      percentageWon = (userWon / turnsPlayed) * 100;
+
+      if (percentageWon >= 50) {
+        return (myOutputValue = `You chose ${inputObject} <br> The computer chose ${outputObject}. <br><br>You lose! Bummer.<br><br>So far ${userName}, you have been winning ${userWon}/${turnsPlayed} turns (or ${percentageWon}%).Still doing well!`);
+      } else if (percentageWon < 50) {
+        return (myOutputValue = `You chose ${inputObject} <br> The computer chose ${outputObject}. <br>You lose! Bummer.<br><br>So far ${userName}, you have been winning ${userWon}/${turnsPlayed} turns (or ${percentageWon}%). Hope your luck turns!`);
```

Comment:
As this code is duplicated for your win conditions, you can create a helper function so that you can write this once and reuse it again